CODE/Sources/PennyBOT/CoinService/DefaultCoinService.swift

```diff
@@ -19,10 +19,10 @@ struct DefaultCoinService: CoinService {
         let data = try JSONEncoder().encode(coinRequest)
         request.body = .bytes(data)
         let response = try await httpClient.execute(request, timeout: .seconds(30), logger: self.logger)
-        logger.trace("HTTP head \(response)")
+        logger.trace("Received HTTP Head", metadata: ["response": "\(response)"])
```

Comment:
I've refined log statements to use `metadata`, mainly so they look much nicer in Discord, but they should also look nicer in stdout logs.